microk8s-resources/actions/enable.kubeflow.sh

```diff
@@ -79,7 +79,7 @@ def main():
         sys.exit(1)
 
     print("Deploying Kubeflow...")
-    juju("bootstrap", "microk8s", "uk8s")
+    juju("bootstrap", "microk8s", "uk8s", "--config", "juju-no-proxy=10.0.0.1")
```

Comment:
What is on 10.0.0.1?

Reply:
In a private discussion with @wallyworld he told me that this IP has to be the IP of the controller running in the MicroK8s cluster. The controller IP is only known after bootstrap. Based on this discussion I have to -1 the PR, we should find another way to get this working.